### PR TITLE
Feat Product Page

### PR DESCRIPTION
**Pull request by Karan-Bharti**
Added product page
image

### CODE REVIEW

File: Products/products.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+.display {
+    display: flex;
+}
+
+.sidebar-heading {
+    font-size: 1.7rem;
+    font-weight: 600;
+    margin: 1rem 0rem;
```
Comment (vaibhav-mahalle) [inline Products/products.css:29]: you can extract margins and padding in the utility classes.

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean good to merge

File: Products/products.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+.sidebar-head {
+    display: flex;
+}
+
+.clear-btn {
+    float: right;
+    display: flex;
```
Comment (vaibhav-mahalle) [inline Products/products.css:38]: use utility classes

File: Products/products.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    justify-content: center;
+    height: 2rem;
```
Comment (vaibhav-mahalle) [inline Products/products.css:40]: same as above

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some changes please look into it.

File: Products/products.css (hunk 1 above)
Comment (Karan-Bharti) [reply Products/products.css:29]: I will keep in mind your suggestions for the next time while coding

**Pull request merged**